src/Symfony/Component/Security/Http/RememberMe/RememberMeHandlerInterface.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\RememberMe;
+
+use Symfony\Component\Security\Core\Exception\AuthenticationException;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * Handles creating and validating remember me cookies.
+ *
+ * If you want to add a custom implementation, you want to extend from
+ * {@see AbstractRememberMeHandler} instead.
+ *
+ * @author Wouter de Jong <wouter@wouterj.nl>
+ *
+ * @experimental in 5.3
+ */
+interface RememberMeHandlerInterface
+{
+    /**
+     * Creates a remember me cookie.
+     *
+     * The actual cookie should be set as an attribute on the master request,
+     * which is transformed into a response cookie by {@see ResponseListener}.
+     */
+    public function createRememberMeCookie(UserInterface $user): void;
+
+    /**
+     * Validates the remember me cookie and returns the associated User.
+     *
+     * This method should also invalidate the current cookie, if this is
+     * stored in a different place (e.g. a database).
```

Comment:
Can you explain this more? Are you referring to the remember me cookie? If so, why would we need to invalidate it? Is this what we're doing in, for example, `PersistentRememberMeHandler::processRememberMe()` when we call `$this->createCookie()`?

Reply:
Updated the PHPdocs